cmd/hyperkube/hyperkube.go

```diff
@@ -55,7 +55,7 @@ func (hk *HyperKube) AddServer(s *Server) {
 // FindServer will find a specific server named name.
 func (hk *HyperKube) FindServer(name string) (*Server, error) {
 	for _, s := range hk.servers {
-		if s.Name() == name {
+		if s.Name() == name || s.AlternativeName == name {
```

Comment:
it's not 100% correct, but I wonder if just additionally checking for a `kube-` prefix would be an easier way of accomplishing the same thing.